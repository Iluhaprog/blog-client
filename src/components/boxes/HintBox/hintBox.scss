@import '../../../vars.scss';

.hint-box {
    height: max-content;
    color: $text-color-5;
    border-left: $border-3;
    padding: 0;
    padding-left: 50px;
    font-size: 14px;
    transition: .2s ease-in-out;
    
    &:hover {
        color: $text-color-6;
    }
}