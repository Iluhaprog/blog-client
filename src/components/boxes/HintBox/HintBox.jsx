import React from 'react';
import './hintBox.scss';

export default ({ children }) => (
    <div className='hint-box'>
        { children }
    </div>
)