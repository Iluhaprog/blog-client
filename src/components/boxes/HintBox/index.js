import HintBox from './HintBox';

export {
    HintBox,
};