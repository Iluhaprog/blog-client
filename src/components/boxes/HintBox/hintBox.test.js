import React from 'react';
import { render } from '@testing-library/react';
import HintBox from './HintBox';

test('Test view of hint box', () => {
    const { asFragment } = render(<HintBox>My hint</HintBox>);
    expect(asFragment()).toMatchSnapshot();
});