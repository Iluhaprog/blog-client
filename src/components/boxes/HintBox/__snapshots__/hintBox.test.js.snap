// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test view of hint box 1`] = `
<DocumentFragment>
  <div
    class="hint-box"
  >
    My hint
  </div>
</DocumentFragment>
`;
